doc/tutorials/modeling-complex-events-ctim.md

```diff
@@ -0,0 +1,77 @@
+
+```
+ {"type": "relationship",
+ "source": "CTIM Modeling Tutorials",
+ "source_uri": "https://github.com/threatgrid/ctim",
+ "source_ref": "indicator-3",
+ "relationship_type": "based-on",
+ "target_ref": "indicator-1"
+ "description": "indicator-3 based-on indicator-1"},
+ {"type": "relationship",
+ "source": "CTIM Modeling Tutorials",
+ "source_uri": "https://github.com/threatgrid/ctim",
+ "source_ref": "indicator-3",
+ "relationship_type": "based-on",
+ "target_ref": "indicator-2"
+ "description": "indicator-3 based-on indicator-2"}
+```
+
+In teams that design and build firewalls and network intrusion detection systems, threat analysts work with rule authors to define these types of patterns ahead of time, and also define the relationships between them. This is, ideally, when the `indicator` entities should be modeled.
+
+Like elsewhere in CTIM, our `indicator` entities do not actually include any details of the actual patterns they detect. We don't ever add YARA rules, or OSQuery details, or any other kind of process-specific details to our `indicator` entities.
+
+The `indicator` entities should contain a brief description of what any given match of the pattern or detection rule might imply.
+- `indicator` entities never contain information about the patterns themselves.
+- `indicator` entities only contain brief or summary overviews of the threat that was detected.
+- Detailed information about the exact nature of the threat is modeled in *threat context entities* such as the `attack-pattern`, `malware`, `tool`, or `vulnerability` entities. Our `indicator` entities can have `indicates` relationships to these. This hybrid normalized data model dramatically reduces the edge complexity of the entity graph (by using indicators as an edge aggregator), and simplifies our ability to update and maintain our detailed threat intelligence. For example, rather than explaining over and over *and over* again in hundreds (or thousands) of `indicator` entities just what exactly "ransomware" is, we can maintain one very well written ransomware `attack-pattern` entity, and link to that from all of those indicators.
+- This summary overview, in the case of a complex pattern match, could also briefly reference the simpler patterns that together triggered the pattern match, but it's not required.
+
+The lesson to learn here is that regardless of how complex your pattern matching or detection rules are, and regardless of whether any one of your rules is composed from a collection of other, similar rules, you can model the information about each pattern matching rule as `indicator` entities in CTIM. Complex rules--where one rule is based on a composition of one or more other rules--are modeled using the `based-on` relationship type.
```

Comment:
`other, similar rules` reads a bit awkwardly to me, maybe drop the comma and/or parenthesize `similar`.